libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -479,6 +445,19 @@ bool AP_BattMonitor::get_temperature(float &temperature, const uint8_t instance)
     }
 }
 
+bool AP_BattMonitor::arming_checks(size_t buflen, char *buffer) const
+{
+    char temp_buffer[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1] {};
+
+    for (uint8_t i = 0; i < AP_BATT_MONITOR_MAX_INSTANCES; i++) {
+        if (drivers[i] != nullptr && !(drivers[i]->arming_checks(sizeof(temp_buffer), temp_buffer))) {
```

Comment:
I'd prefer the `continue;` pattern as used earlier in this patchset...

Reply:
On things that only do one thing here I actually find the early continue a higher load to track so I prefer to not use it in this case. If we get a tie breaker either way I'm happy to change it, otherwise I'm inclined to leave this part alone.